### PR TITLE
Hide completion counts when they would show 0/0 [Followup]

### PR DESCRIPTION
**Pull request by ianlaidlaw**
## Relevant Rally Links

- https://rally1.rallydev.com/#/289692574792d/custom/355050439968?detail=%2Fuserstory%2F613009287361&fdp=true

## Description

This is a follow up to [my previous pr](https://github.com/BrightspaceHypermediaComponents/sequences/pull/345) which missed some spec. It fixes the following things:
1. Unrelated, but Lesson completion counts were not showing due to a previous defect. This fixes that and then hides them if the total is `0/0`
2. Hides `0/0` completion counts for folders

I also went through with a screen reader and ensured that `0/0` completion counts aren't being unexpectedly read out when they would be visually hidden.

### BEFORE:

image

### AFTER:

image

### CODE REVIEW

File: d2l-sequence-navigator/d2l-outer-module.js [hunk 1]
```diff
@@ -601,7 +604,7 @@ class D2LOuterModule extends PolymerASVLaunchMixin(CompletionStatusMixin()) {
 	}
 
 	_hasChildren(entity) {
-		return entity && entity.getSubEntities().length !== 0;
+		return HypermediaHelper.getTopicAndModuleSubEntities(entity).length !== 0;
```
Comment (Kiefer-Co-D2L) [inline d2l-sequence-navigator/d2l-outer-module.js:607]: Question: What's the purpose of this change?

Comment (ianlaidlaw) [reply d2l-sequence-navigator/d2l-outer-module.js:607]: This was the "unrelated" bug I talked about in the description. Basically at some point in the past, another team changed how `EntityBehaviour` worked. If you passed in nothing to `getSubEntities()` it used to mean "give me ALL subentities, regardless of the rel." but they changed it to require a rel or a regexp. When we fixed this bug, I made this helper function that does exactly that. This is just a missed spot that was never updated. Thankfully it was only visual.

Comment (ianlaidlaw) [reply d2l-sequence-navigator/d2l-outer-module.js:607]: https://github.com/BrightspaceHypermediaComponents/sequences/pull/317

**Review by Kiefer-Co-D2L — APPROVED** [review]
